index.js

```diff
@@ -0,0 +1,292 @@
+const NOTES_DOM = {
+    notes_board:document.querySelector("#tasksBoard"),
+    note_name:document.querySelector("#noteName"),
+    note_text:document.querySelector("#noteText"),
+    note_date:document.querySelector("#dateToDo"),
+    note_time:document.querySelector("#timeToDo"),
+    add_button:document.querySelector("#addNoteButton"),
+    reset_button:document.querySelector("#resetFormButton")
+}
+
+NOTES_DOM.add_button.addEventListener("click",saveStickyNOTE)
+
+let arrayOfNotes = [];
+
+function getSelectedValue() {
+let sortByVal = document.querySelector("#sortBy").value
+sortBy(sortByVal , arrayOfNotes)
+}
+
+
+
+
+// drawing functions
+function draw(arr){
+    console.log(arr)
+    clearBoard();
+    for (let index = 0; index < arr.length; index++) {
+      drawNote(arr[index]);
+    }
+
+}
+
+
+function clearBoard(){
+   NOTES_DOM.notes_board.innerHTML =  "";
+}
+function findIndex(arr,id) {
+  
+    for (let index = 0; index < arr.length; index++) {
+      if (arr[index].note_id == id) {
+
+        return index;
+      }
+    }
+}
+
+
+
+function  drawNote(stickyNote){
+   const { notes_board } = NOTES_DOM;
+   const currentNote = getNoteUi(stickyNote)
+   let currentid = currentNote.id.slice(0,10)
+    let currenTime = Date.now().toString().slice(0,10)
+    if (!currentNote) return;
+
+   if (currentid === currenTime){
+    $(currentNote).hide().appendTo(notes_board).fadeIn(1000);
+    
+   }
+   notes_board.append(currentNote);
+    
+}
+
+// main function save sticky note
+function saveStickyNOTE() {
```

Comment:
this function does too much, I would add another function called validate that does the validation